pkg/virtual/workspaces/registry/rest.go

```diff
@@ -254,11 +253,16 @@ func (s *REST) Watch(ctx context.Context, options *metainternal.ListOptions) (wa
 		return nil, fmt.Errorf("no user")
 	}
 
+	orgName, org, err := s.extractOrg(ctx)
```

Comment:
orgClusterName

Reply:
Fixed in commit c48becc80ef541475a0e032a26cfa6a9024c5ac8